frontend/public/components/catalog/_catalog.scss

```diff
@@ -62,7 +62,7 @@ $catalog-tile-width: $co-m-catalog-tile-width;
   }
 
   &__tag {
-    color: $pf-color-black-600;
+    color: var(--pf-global--Color--400);
```

Comment:
Did you intend to use `var(--pf-global--Color--200)` here?

scss variable gets [updated here](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/scss-variables.scss#L45), css var gets [assigned here](https://github.com/patternfly/patternfly/blob/main/src/patternfly/base/_variables.scss#L100)

Reply:
updated